### PR TITLE
[CI:DOCS] Document transient_store in man page

### PR DESCRIPTION
**Pull request by rhatdan**

### CODE REVIEW

**Comment by rhatdan** [comment]
@alexlarsson @nalind @TomSweeneyRedHat @flouthoc @giuseppe @vrothberg @saschagrunert PTAL

File: docs/containers-storage.conf.5.md [hunk 1]
```diff
@@ -64,6 +64,12 @@ Default directory to store all temporary writable content created by container s
 By default, the storage driver is set via the `driver` option. If it is not defined, then the best driver will be picked according to the current platform. This option allows you to override this internal priority list with a custom one to prefer certain drivers.
 Setting this option only has an effect if the local storage has not been initialized yet and the driver name is not set.
 
+**transient_store** = "false" | "true"
+
+Transient store mode makes all container metadata be saved in temporary storage
+(i.e. runroot above). This is faster, but doesn't persist across reboots.
+(Default to false)
```
Comment (nalind) [inline docs/containers-storage.conf.5.md:71]:
```suggestion
Additional garbage collection must also be performed at boot-time, so this option should remain disabled in most configurations. (default: false)
```
There's a "(Default 128k)" at line 181, too, but every other place in this page where we list a default, it's written as "(default: ...)".

Comment (TomSweeneyRedHat) [reply docs/containers-storage.conf.5.md:71]: I think @nalind is suggesting leaving lines 67 through 70 as you have them.  I concur.  I also like his replacement line for line 71.  Add that, then LGTM

**Review by flouthoc — APPROVED** [review]
LGTM

**Pull request merged**